sql3/planner/inbuiltfunctionsdate.go

```diff
@@ -71,6 +73,37 @@ func (p *ExecutionPlanner) analyzeFunctionToTimestamp(call *parser.Call, scope p
 	return call, nil
 }
 
+func (p *ExecutionPlanner) analyzeFunctionDatetimeAdd(call *parser.Call, scope parser.Statement) (parser.Expr, error) {
+	//param1 is the time unit of duration to be added to the target timestamp.
```

Comment:
I really appreciate comments in code!

Just a note that [go doc comments ](https://tip.golang.org/doc/comment) should have a space after the `//`. Comments without the space can sometimes be used as build time instructions.